setup.cfg

```diff
@@ -1,13 +1,64 @@
+[metadata]
+name = pandas
+description = Powerful data structures for data analysis, time series, and statistics
+long_description = file: README.md
+long_description_content_type = text/markdown
+url = https://pandas.pydata.org
+author = The PyData Development Team
+author_email = pydata@googlegroups.com
```

Comment:
While at it, should probably update this to pandas-dev@python.org, and the "author"